res/client/client.css

```
@@ -972,23 +1012,29 @@ QPushButton:disabled
+#resetButton, #searchPlayerButton, #UIButton, #uploadButton, #kickButton, #leaveButton,
+QPushButton#buttonManageInstalled
 {
     color: silver;
     background: #202025;
     border: 1 solid #555450;
     border-radius: 2;
+    padding-left: 2px;
+    padding-right: 2px;
 }
 
 QPushButton#refreshButton:hover, #nextButton::hover, #previousButton::hover, #goToPageButton::hover,
```

Stylelint (17.4.0)

[error] Unexpected unknown pseudo-element selector "::hover" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)